src/components/FilterGroup.tsx

```diff
@@ -51,6 +54,10 @@ export interface FilterGroupProps {
   showMoreLimit?: number
 }
 
+const builtInCssClasses: Readonly<FilterGroupCssClasses> = {
+  titleLabel: 'text-neutral-dark text-sm font-medium text-left mb-4',
```

Comment:
It does. It was being duplicated here for if `collapsible` is `false`

Reply:
if there's a `builtInCssClasses` there too, could we reference that in this file instead of creating another `builtInCssClasses` specially for it here?

Reply:
They are off by 1 so I need a tsMerge to add it back to just the div not CollapsibleLabel

Reply:
closing because discussion moved [here](https://github.com/yext/search-ui-react/pull/321#discussion_r1012319966)